python/jsbsim.pxd

```diff
@@ -30,6 +31,29 @@ cdef extern from "initialization/FGInitialCondition.h" namespace "JSBSim":
         c_FGInitialCondition(c_FGInitialCondition* ic)
         bool Load(const c_SGPath& rstfile, bool useStoredPath)
 
+cdef extern from "initialization/FGLinearization.h" namespace "JSBSim":
+    cdef cppclass c_FGLinearization "JSBSim::FGLinearization":
+        c_FGLinearization(c_FGFDMExec* fdme)
+        
+        void WriteScicoslab() const
+        void WriteScicoslab(string& path) const
+
+        void GetStateSpace(vector[vector[double]]& A_, vector[vector[double]]& B_,
+                           vector[vector[double]]& C_, vector[vector[double]]& D_) const
```

Comment:
Replace this method with 4 separate and more explicitly named methods `GetSystemMatrix`, `GetInputMatrix`, `GetOutputMatrix` and `GetFeedforwardMatrix` as suggested in the header `FGLinearization.h` below.
```suggestion
        vector[vector[double]]& GetSystemMatrix() const
        vector[vector[double]]& GetInputMatrix() const
        vector[vector[double]]& GetOutputMatrix() const
        vector[vector[double]]& GetFeedforwardMatrix() const
```